modules/admin/admin-impl/src/test/resources/com/enonic/xp/admin/impl/tool/XmlAdminToolDescriptorParserTest-interfaces.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<tool xmlns="urn:enonic:xp:model:1.0">
+  <display-name i18n="key.display-name">My admin tool</display-name>
+  <description i18n="key.description">Tool description</description>
+  <allow>
+    <principal>role:system.admin</principal>
+  </allow>
+  <interfaces>
+    <interface>generic</interface>
```

Comment:
A bit confusing even in test to specify generic interface in admin tool

Reply:
It is a just test of parser :)